README.md

```diff
@@ -12,12 +12,28 @@ It is also a Stranger Things reference.
 Picture of Eleven
 -->
 
-## Install
+### Install
 
 ```
 npm i undici
 ```
 
+### Benchmarks
+
+Machine: Intel i7-7700k, 4.5GHz, 4C/8T, 32GB RAM, NVMe SSD\
+Server: [H2O](https://github.com/h2o/h2o) 2.2.5\
+Configuration: HTTP/1.1 without TLS
+
+```
+http - keepalive - read x 17,074 ops/sec ±6.99% (79 runs sampled)
+undici - request - read x 25,428 ops/sec ±1.73% (83 runs sampled)
+
+http - keepalive - pipe x 15,504 ops/sec ±7.32% (79 runs sampled)
+undici - stream  - pipe x 24,073 ops/sec ±1.86% (88 runs sampled)
+```
```

Comment:
Why 4 and not 3? It's a bit confusing to have http twice but with different results. :/

Reply:
Looks like stream is slower than request

Reply:
Ah, `pipe` vs `read`.

Reply:
I would just do the pipe version for all 3

Reply:
It's different usage. `pipe` does `stream.pipe(anotherStream)`, while `read` does `stream.resume(); stream.once('end', () => deffered.resolve())`.

Reply:
Ah, for some reason I had to refresh to get your comments. Will take another look at this.

Reply:
> Looks like stream is slower than request

That's just an error. `±1.73%` vs `±1.86%`. It's basically the same.

Reply:
But is read relevant in the real world? Seems a bit synthetic. I’m fine with it. Just thinking out loud.

Reply:
Hmm... Honestly I'd prefer all four, but I don't mind having only three. @mcollina Any thoughts?

Reply:
> But is read relevant in the real world? Seems a bit synthetic.

Nah, you're right. It **is** synthetic. Let me fix that.